### PR TITLE
Fix test to avoid extra failures during development

### PR DESCRIPTION
**Pull request by babenek**
## Description

- Fix test to avoid extra failures during development from extra .json files as old artifacts

## How has this been tested?

- [x] create any json file in root of repo and run project (main branch fails)

### CODE REVIEW

**Review by csh519 — APPROVED** [review]
Avoiding test error is good!

And I think it would be good to increase fuzzing coverage in the future to fit with new calculate method.

LGTM :+1:

**Review by kmnls — DISMISSED** [review]

File: .github/workflows/fuzz.yml [hunk 1]
```diff
@@ -78,7 +78,7 @@ jobs:
       run: |
         COVERAGE=$(tail -1 report.txt | awk '{print $6}' | tr --delete '%')
         # additionally check correctness of the value - should be an integer
-        if ! [ 75 -le ${COVERAGE} ]; then
+        if ! [ 74 -le ${COVERAGE} ]; then
```
Comment (babenek) [inline .github/workflows/fuzz.yml:81]:
```suggestion
        if ! [ 75 -le ${COVERAGE} ]; then
```

**Review by iuriimet — APPROVED** [review]
Looks good

**Review by csh519 — APPROVED** [review]
Approve again.

**Pull request merged**